src/basics.php

```diff
@@ -120,6 +120,20 @@ function breakpoint(): ?string
     }
 }
 
+if (!function_exists('table') && (PHP_SAPI === 'cli' || PHP_SAPI === 'phpdbg') ) {
+    /**
+     * REPL convenience function to load a table
+     *
+     * @param string $alias The alias name you want to get.
+     * @param array $options The options you want to build the table with.
+     * @return \Cake\ORM\Table
+     */
+    function table($alias, array $options = [])
```

Comment:
This might better go into 4.next
=> why not stict typehing here for string $alias?

Reply:
> => why not stict typehing here for string $alias?

Simply an oversight because the code started in a 3.x branch. Fixed.

Reply:
> This might better go into 4.next

Changed and rebased.